server/protocols/jmap/src/main/java/org/apache/james/jmap/JMAPConfiguration.java

```diff
@@ -84,6 +86,11 @@ public JMAPConfiguration build() {
             Preconditions.checkState(!enabled.get() || !Strings.isNullOrEmpty(secret), "'secret' is mandatory");
             return new JMAPConfiguration(enabled.get(), keystore, secret, jwtPublicKeyPem, port);
         }
+
+        private boolean validPublicKey(Optional<String> jwtPublicKeyPem) {
+            PublicKeyReader reader = new PublicKeyReader();
+            return !jwtPublicKeyPem.isPresent() || reader.fromPEM(jwtPublicKeyPem).isPresent();
```

Comment:
You have two checks here:
 - Does the JWT key is valid? -> this one belongs in JWTConfiguration.java
 - JMAP requires a JWT key -> this check should be performed in Jmap configuration